spec/migrations/20140218232357_add_group_and_user_columns_to_miq_widget_contents_spec.rb

```diff
@@ -0,0 +1,24 @@
+require_migration
+
+describe AddGroupAndUserColumnsToMiqWidgetContents do
+  shared_examples "removing widget contents" do
+    let(:miq_widget_content_stub) { migration_stub(:MiqWidgetContent) }
+
+    it "removes all existing widget content" do
+      miq_widget_content_stub.create!
+      miq_widget_content_stub.create!
+
+      migrate
+
+      expect(miq_widget_content_stub.all).to be_empty
```

Comment:
Is it ok that we delete all widget content on both up/down in this migration?  I know this migration is old but it seems scary.  Maybe we had no choice.  😕 

I'm fine with this spec change anyway, just wondering if we need to be concerned with this.